src/main/java/com/andrzejewski/todolist/user/UserService.java

```diff
@@ -25,14 +25,14 @@ public UserEntity getById(Long id) {
     }
 
     public boolean checkUsername(String username) {
```

Comment:
You should call this method so as it provides more insights into the return value. Currently, by reading method name you do not know if it returns true if the username is already taken or if it is free. Use something like `isUsernameTaken` or `isUsernameAvaliable`